matRad_interp3.m

```diff
@@ -0,0 +1,49 @@
+function y = matRad_interp3(xi,yi,zi,x,xq,yq,zq,mode,extrapVal)
+% interpolates 3-D data (table lookup) 
+%
+% call
+%   y = matRad_interp3(xi,yi,zi,x,xq,yq,zy)
+%
+% input
+%   xi,yi,zi: grid vectors 
+%   x:        data
+%   xq,yq,zq: coordinates of quer points as a grid
+%	
+% output
+%   y: interpolated data   
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2019 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+persistent env;
```

Comment:
This is not necessary, matRad_getEnvironment() already uses persistant variables to reduce overhead. only line 31 is needed.